Resources/translations/SonataAdminBundle.pl.xliff

```diff
@@ -88,7 +88,7 @@
             </trans-unit>
             <trans-unit id="title_show">
                 <source>title_show</source>
-                <target>title_show</target>
+                <target>Szczegóły</target>
```

Comment:
Please check my changes.
I've changed my mind because in polish its sounds better without the translation of the word "Show". I think that the better option is to leave this parameter alone.